docs/core/compatibility/sdk/8.0/runtimespecific-app-default.md

```diff
@@ -0,0 +1,74 @@
+
+This is a breaking change in the following situations:
+
+- If you deployed, distributed, or published your app and didn't explicitly add the `SelfContained` property, but also didn't require that the .NET runtime be installed on the machine for it to work. In this case, you may have relied on the previous behavior to produce a non-framework-dependent app by default.
+
+- If you rely on the IL Link tool. In this case, you'll have to take the steps described under [Recommended action](#recommended-action) to use IL Link again.
+
+Blazor WebAssembly apps relied on the previous behavior. However, the Blazor SDK now adds `SelfContained` automatically for all apps, so Blazor customers shouldn't be affected.
+
+## Previous behavior
+
+Previously, if a runtime identifier (RID) was specified (via [RuntimeIdentifier](../../../project-sdk/msbuild-props.md#runtimeidentifier)), the app was published as self-contained, even if `SelfContained` wasn't explicitly specified.
+
+In addition:
+
+- If `PublishSelfContained` wasn't explicitly set to `false`, the publish properties `PublishSingleFile` and `PublishAot` implied `PublishSelfContained` and therefore `SelfContained` during operations including `dotnet build`, `dotnet restore`, and `dotnet publish`.
+- The `PublishTrimmed` property did not imply `SelfContained`.
+- The `PublishReadyToRun` property implied `SelfContained` if `SelfContained` wasn't specified.
+
+## New behavior
+
+Starting in .NET 8, for apps that target .NET 8 or a later version, `RuntimeIdentifier` no longer implies `SelfContained` by default. Instead, apps that specify a runtime identifier will be dependent on the .NET runtime by default (framework-dependent). Apps that target .NET 7 or earlier versions aren't affected.
+
+In addition:
+
+- If `PublishSelfContained` isn't explicitly set to `false`, the publish properties `PublishTrimmed`, `PublishSingleFile`, and `PublishAot` now imply `SelfContained` during `dotnet publish` only (that is, not for `dotnet build` or `dotnet restore`).
+- The `PublishTrimmed` property also now implies `SelfContained` during `dotnet publish`.
+- The `PublishReadyToRun` property no longer implies `SelfContained` if the project targets .NET 8 or later.
+
+> [!NOTE]
+> If you publish using `msbuild t:/Publish`, the publish properties don't imply `SelfContained`.
+
+## Version introduced
+
+.NET 8 Preview 5
+
+## Type of breaking change
+
+This change can affect [source compatibility](../../categories.md#source-compatibility) and [binary compatibility](../../categories.md#binary-compatibility).
+
+## Reason for change
+
+- The new .NET SDK behavior aligns with Visual Studio behavior.
+- Framework-dependent apps are now smaller by default, since there aren't copies of .NET stored in each app.
```

Comment:
```suggestion
- Framework-dependent apps are smaller by default, since there aren't copies of .NET stored in each app.
```

We didn't improve FDD apps size in any meaningful way in this release, this should just describe the difference between SCD and FDD.

Reply:
Do you think it would be better to reword the benefit of this? I didn't communicate it super well. It is still true that it saves space on user's machines. If more apps are FDD because it's the default, then there will be less copies of .NET and therefore more space on the machine.

Reply:
For me personally just dropping the "now" fixes the meaning, but I'm not a native speaker, so I can't tell for sure.